response.go

```diff
@@ -56,7 +82,7 @@ func PreloadableResponse(resp *http.Response) (header *string, issues Issues) {
 // To interpret `issues`, see the list of conventions in the
 // documentation for Issues.
 func RemovableResponse(resp *http.Response) (header *string, issues Issues) {
-	return checkResponse(resp, RemovableHeaderString)
+	return checkResponse(resp, RemovableHeaderStringWrapper, preloadlist.Bulk1Year)
```

Comment:
`RemovableHeaderStringWrapper` shouldn't be part of the exported API for this package. Instead, you can make an anonymous function to pass in here, e.g. `func(domain string, policy preloadlist.PolicyType) Issues { return RemovableHeaderString(domain) }`.